src/stats_tracker.cpp

```diff
@@ -66,6 +66,54 @@ int event_multiset::total( const std::string &field, const cata::event::data_typ
     return total;
 }
 
+int event_multiset::minimum(const std::string& field) const
+{
+    return minimum(field, {});
+}
+
+int event_multiset::minimum(const std::string& field, const cata::event::data_type& criteria) const
+{
+    int minimum = 0;
```

Comment:
I'm trying to decide if it's reasonable to always assume that 0 is a good default value.  We *could* have a default passed to us, but I guess YAGNI so leave it for now.

Reply:
Yeah, I contemplated the same but didn't have something with a non-zero default to exercise it and so left it with zero.